scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=5650
-export JSHINT_THRESHOLD=9390
+export JSHINT_THRESHOLD=9080
```

Comment:
@andy-armstrong The error wouldn't be thrown if it was defined with RequireJS right?